scripts/prepare_vdpa.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -x
+
+sudo apt install -y libncurses-dev gawk flex bison openssl libssl-dev dkms libelf-dev libudev-dev libpci-dev libiberty-dev autoconf git make dpkg-dev libmnl-dev pkg-config
+sudo sed -i -- 's/# deb-src/deb-src/g' /etc/apt/sources.list
+sudo apt update
+apt-get source linux-image-unsigned-`uname -r`
+pushd linux-azure*/drivers/vdpa/vdpa_sim/
+cat <<'EOF' > Makefile
+# SPDX-License-Identifier: GPL-2.0
+obj-m += vdpa_sim.o
+obj-m += vdpa_sim_net.o
+obj-m += vdpa_sim_blk.o
+EOF
+make -C /lib/modules/`uname -r`/build M=$PWD
+sudo make -C /lib/modules/`uname -r`/build M=$PWD modules_install
+popd
+sudo depmod -a
+sudo modprobe vdpa
+sudo modprobe vhost_vdpa
+sudo modprobe vdpa_sim
+sudo modprobe vdpa_sim_blk
+sudo modprobe vdpa_sim_net
+git clone https://github.com/shemminger/iproute2.git
```

Comment:
If you can wait for us to move to jammy for the container the binary is already present: https://packages.ubuntu.com/search?suite=jammy&arch=any&mode=exactfilename&searchon=contents&keywords=vdpa

Reply:
Well it's up to you, but I can remove it later when we move to Jammy. I mean the download/build/install of `iproute2` is super fast so I'm not too worried.

Reply:
Please open a GH issue to track this.

Reply:
If you actually mention it in a github comment in the code I think it might cross-reference back but at the very least it is easy to search for.

Reply:
Dittor

Reply:
Done!